ansible_collections/netapp/ontap/plugins/modules/na_ontap_info.py

```diff
@@ -429,7 +429,7 @@ def __init__(self, module, max_records):
                 'method': self.get_generic_get_iter,
                 'kwargs': {
                     'call': 'cluster-image-get-iter',
-                    'attribute': 'cluster-image-info',
+                    'attribute': 'node-id',
```

Comment:
are you sure about this?
I'd think we need 
```
                    'call': 'cluster-image-get-iter',
                    'attribute': 'cluster-image-info',
                    'key_fields': 'node-name',
                    'query': {'max-records': self.max_records},
```

Reply:
oh, you're right of course. I have not idea how I made this incorrect push but in my current installation I have 
```
                    'attribute': 'cluster-image-info',
                    'key_fields': 'node-id',
```